changeset:   260:c1745dec392a
tag:         tip
user:        Guillermo Alvarez de Eulate <guillermo.alvarez@openbravo.com>
date:        Fri Dec 22 11:52:20 2017 +0100
summary:     Increase stock for AVALANCHE TRANSCEIVER - Related to SCO test merge

diff -r 42d9672ab154 -r c1745dec392a referencedata/sampledata/The_White_Valley_Group/M_INVENTORY.xml
--- a/referencedata/sampledata/The_White_Valley_Group/M_INVENTORY.xml	Wed Dec 20 09:50:53 2017 +0100
+++ b/referencedata/sampledata/The_White_Valley_Group/M_INVENTORY.xml	Fri Dec 22 11:52:20 2017 +0100
@@ -111,4 +111,26 @@
   <EM_OBMWHPI_PROCESSING><![CDATA[N]]></EM_OBMWHPI_PROCESSING>
 </M_INVENTORY>
 
+<M_INVENTORY>
+  <M_INVENTORY_ID><![CDATA[780AF9BCD96947D981C8BAD7160C4BA8]]></M_INVENTORY_ID>
+  <AD_CLIENT_ID><![CDATA[39363B0921BB4293B48383844325E84C]]></AD_CLIENT_ID>
+  <AD_ORG_ID><![CDATA[D270A5AC50874F8BA67A88EE977F8E3B]]></AD_ORG_ID>
+  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+  <CREATED><![CDATA[2017-12-22 10:53:45.245]]></CREATED>
+  <CREATEDBY><![CDATA[100]]></CREATEDBY>
+  <UPDATED><![CDATA[2017-12-22 10:56:49.586843]]></UPDATED>
+  <UPDATEDBY><![CDATA[100]]></UPDATEDBY>
+  <NAME><![CDATA[22-12-2017 09:53:33]]></NAME>
+  <M_WAREHOUSE_ID><![CDATA[CE7AB151695342FF879938F837F12E6E]]></M_WAREHOUSE_ID>
+  <MOVEMENTDATE><![CDATA[2017-12-22 00:00:00.0]]></MOVEMENTDATE>
+  <POSTED><![CDATA[Y]]></POSTED>
+  <PROCESSED><![CDATA[Y]]></PROCESSED>
+  <PROCESSING><![CDATA[N]]></PROCESSING>
+  <UPDATEQTY><![CDATA[N]]></UPDATEQTY>
+  <GENERATELIST><![CDATA[N]]></GENERATELIST>
+  <INVENTORY_TYPE><![CDATA[N]]></INVENTORY_TYPE>
+  <EM_OBMWHPI_STATUS><![CDATA[P]]></EM_OBMWHPI_STATUS>
+  <EM_OBMWHPI_PROCESSING><![CDATA[N]]></EM_OBMWHPI_PROCESSING>
+</M_INVENTORY>
+
 </data>
diff -r 42d9672ab154 -r c1745dec392a referencedata/sampledata/The_White_Valley_Group/M_INVENTORYLINE.xml
--- a/referencedata/sampledata/The_White_Valley_Group/M_INVENTORYLINE.xml	Wed Dec 20 09:50:53 2017 +0100
+++ b/referencedata/sampledata/The_White_Valley_Group/M_INVENTORYLINE.xml	Fri Dec 22 11:52:20 2017 +0100
@@ -61,6 +61,25 @@
 </M_INVENTORYLINE>
 
 <M_INVENTORYLINE>
+  <M_INVENTORYLINE_ID><![CDATA[59C5AB5183D244E587F530C342E3DA77]]></M_INVENTORYLINE_ID>
+  <AD_CLIENT_ID><![CDATA[39363B0921BB4293B48383844325E84C]]></AD_CLIENT_ID>
+  <AD_ORG_ID><![CDATA[D270A5AC50874F8BA67A88EE977F8E3B]]></AD_ORG_ID>
+  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+  <CREATED><![CDATA[2017-12-22 10:54:04.7]]></CREATED>
+  <CREATEDBY><![CDATA[100]]></CREATEDBY>
+  <UPDATED><![CDATA[2017-12-22 10:54:04.7]]></UPDATED>
+  <UPDATEDBY><![CDATA[100]]></UPDATEDBY>
+  <M_INVENTORY_ID><![CDATA[780AF9BCD96947D981C8BAD7160C4BA8]]></M_INVENTORY_ID>
+  <M_LOCATOR_ID><![CDATA[FDC4AC6020F841B3BAA73048C5EA7180]]></M_LOCATOR_ID>
+  <M_PRODUCT_ID><![CDATA[934E7D7587EC4C7A9E9FF58F0382D450]]></M_PRODUCT_ID>
+  <LINE><![CDATA[10]]></LINE>
+  <QTYBOOK><![CDATA[62]]></QTYBOOK>
+  <QTYCOUNT><![CDATA[500]]></QTYCOUNT>
+  <C_UOM_ID><![CDATA[100]]></C_UOM_ID>
+  <EM_OBMWHPI_COUNTED><![CDATA[Y]]></EM_OBMWHPI_COUNTED>
+</M_INVENTORYLINE>
+
+<M_INVENTORYLINE>
   <M_INVENTORYLINE_ID><![CDATA[6FDF9218541C4321BA90A280A42013E0]]></M_INVENTORYLINE_ID>
   <AD_CLIENT_ID><![CDATA[39363B0921BB4293B48383844325E84C]]></AD_CLIENT_ID>
   <AD_ORG_ID><![CDATA[E41B10B390B0430289A00FD6E5CC0CDA]]></AD_ORG_ID>
diff -r 42d9672ab154 -r c1745dec392a referencedata/sampledata/The_White_Valley_Group/M_STORAGE_DETAIL.xml
--- a/referencedata/sampledata/The_White_Valley_Group/M_STORAGE_DETAIL.xml	Wed Dec 20 09:50:53 2017 +0100
+++ b/referencedata/sampledata/The_White_Valley_Group/M_STORAGE_DETAIL.xml	Fri Dec 22 11:52:20 2017 +0100
@@ -310,15 +310,15 @@
   <M_LOCATOR_ID><![CDATA[FDC4AC6020F841B3BAA73048C5EA7180]]></M_LOCATOR_ID>
   <M_ATTRIBUTESETINSTANCE_ID><![CDATA[0]]></M_ATTRIBUTESETINSTANCE_ID>
   <C_UOM_ID><![CDATA[100]]></C_UOM_ID>
-  <QTYONHAND><![CDATA[1000]]></QTYONHAND>
+  <QTYONHAND><![CDATA[999]]></QTYONHAND>
   <PREQTYONHAND><![CDATA[0]]></PREQTYONHAND>
   <AD_CLIENT_ID><![CDATA[39363B0921BB4293B48383844325E84C]]></AD_CLIENT_ID>
   <AD_ORG_ID><![CDATA[BF129721D9FB4EB0819509934153E972]]></AD_ORG_ID>
   <ISACTIVE><![CDATA[Y]]></ISACTIVE>
   <CREATED><![CDATA[2013-07-04 23:01:14.894]]></CREATED>
   <CREATEDBY><![CDATA[0]]></CREATEDBY>
-  <UPDATED><![CDATA[2013-07-04 23:01:14.894]]></UPDATED>
-  <UPDATEDBY><![CDATA[0]]></UPDATEDBY>
+  <UPDATED><![CDATA[2017-12-22 11:37:58.0]]></UPDATED>
+  <UPDATEDBY><![CDATA[3073EDF96A3C42CC86C7069E379522D2]]></UPDATEDBY>
   <M_STORAGE_DETAIL_ID><![CDATA[0D5DFCE7B6B94D1B877FE2FAD7E8E270]]></M_STORAGE_DETAIL_ID>
   <RESERVEDQTY><![CDATA[0]]></RESERVEDQTY>
   <ALLOCATEDQTY><![CDATA[0]]></ALLOCATEDQTY>
@@ -4957,15 +4957,16 @@
   <M_LOCATOR_ID><![CDATA[FDC4AC6020F841B3BAA73048C5EA7180]]></M_LOCATOR_ID>
   <M_ATTRIBUTESETINSTANCE_ID><![CDATA[0]]></M_ATTRIBUTESETINSTANCE_ID>
   <C_UOM_ID><![CDATA[100]]></C_UOM_ID>
-  <QTYONHAND><![CDATA[62]]></QTYONHAND>
+  <QTYONHAND><![CDATA[500]]></QTYONHAND>
+  <DATELASTINVENTORY><![CDATA[2017-12-22 00:00:00.0]]></DATELASTINVENTORY>
   <PREQTYONHAND><![CDATA[-2]]></PREQTYONHAND>
   <AD_CLIENT_ID><![CDATA[39363B0921BB4293B48383844325E84C]]></AD_CLIENT_ID>
   <AD_ORG_ID><![CDATA[D270A5AC50874F8BA67A88EE977F8E3B]]></AD_ORG_ID>
   <ISACTIVE><![CDATA[Y]]></ISACTIVE>
   <CREATED><![CDATA[2013-07-04 23:01:14.882]]></CREATED>
   <CREATEDBY><![CDATA[0]]></CREATEDBY>
-  <UPDATED><![CDATA[2013-07-04 23:01:14.882]]></UPDATED>
-  <UPDATEDBY><![CDATA[0]]></UPDATEDBY>
+  <UPDATED><![CDATA[2017-12-22 10:54:30.0]]></UPDATED>
+  <UPDATEDBY><![CDATA[100]]></UPDATEDBY>
   <M_STORAGE_DETAIL_ID><![CDATA[B4D1E79C095C4DD28220871446629740]]></M_STORAGE_DETAIL_ID>
   <RESERVEDQTY><![CDATA[0]]></RESERVEDQTY>
   <ALLOCATEDQTY><![CDATA[0]]></ALLOCATEDQTY>
diff -r 42d9672ab154 -r c1745dec392a referencedata/sampledata/The_White_Valley_Group/M_TRANSACTION.xml
--- a/referencedata/sampledata/The_White_Valley_Group/M_TRANSACTION.xml	Wed Dec 20 09:50:53 2017 +0100
+++ b/referencedata/sampledata/The_White_Valley_Group/M_TRANSACTION.xml	Fri Dec 22 11:52:20 2017 +0100
@@ -2251,6 +2251,36 @@
 </M_TRANSACTION>
 
 <M_TRANSACTION>
+  <M_TRANSACTION_ID><![CDATA[2CAF73E64E7F427494E9540CFEF8D2BA]]></M_TRANSACTION_ID>
+  <AD_CLIENT_ID><![CDATA[39363B0921BB4293B48383844325E84C]]></AD_CLIENT_ID>
+  <AD_ORG_ID><![CDATA[D270A5AC50874F8BA67A88EE977F8E3B]]></AD_ORG_ID>
+  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+  <CREATED><![CDATA[2017-12-22 10:54:30.593493]]></CREATED>
+  <CREATEDBY><![CDATA[100]]></CREATEDBY>
+  <UPDATED><![CDATA[2017-12-22 10:55:05.762]]></UPDATED>
+  <UPDATEDBY><![CDATA[100]]></UPDATEDBY>
+  <MOVEMENTTYPE><![CDATA[I+]]></MOVEMENTTYPE>
+  <M_LOCATOR_ID><![CDATA[FDC4AC6020F841B3BAA73048C5EA7180]]></M_LOCATOR_ID>
+  <M_PRODUCT_ID><![CDATA[934E7D7587EC4C7A9E9FF58F0382D450]]></M_PRODUCT_ID>
+  <MOVEMENTDATE><![CDATA[2017-12-22 00:00:00.0]]></MOVEMENTDATE>
+  <MOVEMENTQTY><![CDATA[438]]></MOVEMENTQTY>
+  <M_INVENTORYLINE_ID><![CDATA[59C5AB5183D244E587F530C342E3DA77]]></M_INVENTORYLINE_ID>
+  <M_ATTRIBUTESETINSTANCE_ID><![CDATA[0]]></M_ATTRIBUTESETINSTANCE_ID>
+  <C_UOM_ID><![CDATA[100]]></C_UOM_ID>
+  <TRANSACTIONCOST><![CDATA[46563.78]]></TRANSACTIONCOST>
+  <TRXPROCESSDATE><![CDATA[2017-12-22 10:54:30.0]]></TRXPROCESSDATE>
+  <M_COSTING_ALGORITHM_ID><![CDATA[B069080A0AE149A79CF1FA0E24F16AB6]]></M_COSTING_ALGORITHM_ID>
+  <ISCOSTCALCULATED><![CDATA[Y]]></ISCOSTCALCULATED>
+  <C_CURRENCY_ID><![CDATA[102]]></C_CURRENCY_ID>
+  <COSTING_STATUS><![CDATA[CC]]></COSTING_STATUS>
+  <CHECKRESERVEDQTY><![CDATA[Y]]></CHECKRESERVEDQTY>
+  <ISPROCESSED><![CDATA[Y]]></ISPROCESSED>
+  <CHECKPRICEDIFFERENCE><![CDATA[N]]></CHECKPRICEDIFFERENCE>
+  <MANUALCOSTADJUSTMENT><![CDATA[N]]></MANUALCOSTADJUSTMENT>
+  <ISCOSTPERMANENT><![CDATA[N]]></ISCOSTPERMANENT>
+</M_TRANSACTION>
+
+<M_TRANSACTION>
   <M_TRANSACTION_ID><![CDATA[2F23258357BF4145A67B3C52F8E4A993]]></M_TRANSACTION_ID>
   <AD_CLIENT_ID><![CDATA[39363B0921BB4293B48383844325E84C]]></AD_CLIENT_ID>
   <AD_ORG_ID><![CDATA[D270A5AC50874F8BA67A88EE977F8E3B]]></AD_ORG_ID>
@@ -11821,6 +11851,36 @@
 </M_TRANSACTION>
 
 <M_TRANSACTION>
+  <M_TRANSACTION_ID><![CDATA[F4BDC4F0FF5B3A21B61ACAC96A196B92]]></M_TRANSACTION_ID>
+  <AD_CLIENT_ID><![CDATA[39363B0921BB4293B48383844325E84C]]></AD_CLIENT_ID>
+  <AD_ORG_ID><![CDATA[D270A5AC50874F8BA67A88EE977F8E3B]]></AD_ORG_ID>
+  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+  <CREATED><![CDATA[2017-12-22 11:37:59.242]]></CREATED>
+  <CREATEDBY><![CDATA[3073EDF96A3C42CC86C7069E379522D2]]></CREATEDBY>
+  <UPDATED><![CDATA[2017-12-22 11:38:05.337]]></UPDATED>
+  <UPDATEDBY><![CDATA[100]]></UPDATEDBY>
+  <MOVEMENTTYPE><![CDATA[C-]]></MOVEMENTTYPE>
+  <M_LOCATOR_ID><![CDATA[FDC4AC6020F841B3BAA73048C5EA7180]]></M_LOCATOR_ID>
+  <M_PRODUCT_ID><![CDATA[C80062135FF7410DAC9924F5030953AC]]></M_PRODUCT_ID>
+  <MOVEMENTDATE><![CDATA[2017-12-22 00:00:00.0]]></MOVEMENTDATE>
+  <MOVEMENTQTY><![CDATA[-1]]></MOVEMENTQTY>
+  <M_INOUTLINE_ID><![CDATA[F4BDC4F0FF5B3A21B61ACAC96A196B92]]></M_INOUTLINE_ID>
+  <M_ATTRIBUTESETINSTANCE_ID><![CDATA[0]]></M_ATTRIBUTESETINSTANCE_ID>
+  <C_UOM_ID><![CDATA[100]]></C_UOM_ID>
+  <TRANSACTIONCOST><![CDATA[0.93]]></TRANSACTIONCOST>
+  <TRXPROCESSDATE><![CDATA[2017-12-22 11:37:59.2]]></TRXPROCESSDATE>
+  <M_COSTING_ALGORITHM_ID><![CDATA[B069080A0AE149A79CF1FA0E24F16AB6]]></M_COSTING_ALGORITHM_ID>
+  <ISCOSTCALCULATED><![CDATA[Y]]></ISCOSTCALCULATED>
+  <C_CURRENCY_ID><![CDATA[102]]></C_CURRENCY_ID>
+  <COSTING_STATUS><![CDATA[CC]]></COSTING_STATUS>
+  <CHECKRESERVEDQTY><![CDATA[Y]]></CHECKRESERVEDQTY>
+  <ISPROCESSED><![CDATA[Y]]></ISPROCESSED>
+  <CHECKPRICEDIFFERENCE><![CDATA[N]]></CHECKPRICEDIFFERENCE>
+  <MANUALCOSTADJUSTMENT><![CDATA[N]]></MANUALCOSTADJUSTMENT>
+  <ISCOSTPERMANENT><![CDATA[N]]></ISCOSTPERMANENT>
+</M_TRANSACTION>
+
+<M_TRANSACTION>
   <M_TRANSACTION_ID><![CDATA[F53659FFD520488AB922EE00CD3FEA86]]></M_TRANSACTION_ID>
   <AD_CLIENT_ID><![CDATA[39363B0921BB4293B48383844325E84C]]></AD_CLIENT_ID>
   <AD_ORG_ID><![CDATA[BF129721D9FB4EB0819509934153E972]]></AD_ORG_ID>
diff -r 42d9672ab154 -r c1745dec392a referencedata/sampledata/The_White_Valley_Group/M_TRANSACTION_COST.xml
--- a/referencedata/sampledata/The_White_Valley_Group/M_TRANSACTION_COST.xml	Wed Dec 20 09:50:53 2017 +0100
+++ b/referencedata/sampledata/The_White_Valley_Group/M_TRANSACTION_COST.xml	Fri Dec 22 11:52:20 2017 +0100
@@ -613,6 +613,23 @@
 </M_TRANSACTION_COST>
 
 <M_TRANSACTION_COST>
+  <M_TRANSACTION_COST_ID><![CDATA[167D76C6882F47D7A1937D965709C4B6]]></M_TRANSACTION_COST_ID>
+  <AD_CLIENT_ID><![CDATA[39363B0921BB4293B48383844325E84C]]></AD_CLIENT_ID>
+  <AD_ORG_ID><![CDATA[D270A5AC50874F8BA67A88EE977F8E3B]]></AD_ORG_ID>
+  <CREATED><![CDATA[2017-12-22 11:38:05.15]]></CREATED>
+  <CREATEDBY><![CDATA[100]]></CREATEDBY>
+  <UPDATED><![CDATA[2017-12-22 11:38:05.15]]></UPDATED>
+  <UPDATEDBY><![CDATA[100]]></UPDATEDBY>
+  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+  <M_TRANSACTION_ID><![CDATA[F4BDC4F0FF5B3A21B61ACAC96A196B92]]></M_TRANSACTION_ID>
+  <COST><![CDATA[0.93]]></COST>
+  <COSTDATE><![CDATA[2017-12-22 11:37:59.2]]></COSTDATE>
+  <C_CURRENCY_ID><![CDATA[102]]></C_CURRENCY_ID>
+  <ISUNITCOST><![CDATA[Y]]></ISUNITCOST>
+  <DATEACCT><![CDATA[2017-12-22 00:00:00.0]]></DATEACCT>
+</M_TRANSACTION_COST>
+
+<M_TRANSACTION_COST>
   <M_TRANSACTION_COST_ID><![CDATA[17020951932044F1BB7D7C0AE146780E]]></M_TRANSACTION_COST_ID>
   <AD_CLIENT_ID><![CDATA[39363B0921BB4293B48383844325E84C]]></AD_CLIENT_ID>
   <AD_ORG_ID><![CDATA[01AD882EFC8545ACA6455E2F6FD51EE9]]></AD_ORG_ID>
@@ -2976,6 +2993,23 @@
 </M_TRANSACTION_COST>
 
 <M_TRANSACTION_COST>
+  <M_TRANSACTION_COST_ID><![CDATA[64BACF79976D487C93597644F9FE79FB]]></M_TRANSACTION_COST_ID>
+  <AD_CLIENT_ID><![CDATA[39363B0921BB4293B48383844325E84C]]></AD_CLIENT_ID>
+  <AD_ORG_ID><![CDATA[D270A5AC50874F8BA67A88EE977F8E3B]]></AD_ORG_ID>
+  <CREATED><![CDATA[2017-12-22 10:55:05.275]]></CREATED>
+  <CREATEDBY><![CDATA[100]]></CREATEDBY>
+  <UPDATED><![CDATA[2017-12-22 10:55:05.275]]></UPDATED>
+  <UPDATEDBY><![CDATA[100]]></UPDATEDBY>
+  <ISACTIVE><![CDATA[Y]]></ISACTIVE>
+  <M_TRANSACTION_ID><![CDATA[2CAF73E64E7F427494E9540CFEF8D2BA]]></M_TRANSACTION_ID>
+  <COST><![CDATA[46563.78]]></COST>
+  <COSTDATE><![CDATA[2017-12-22 10:54:30.0]]></COSTDATE>
+  <C_CURRENCY_ID><![CDATA[102]]></C_CURRENCY_ID>
+  <ISUNITCOST><![CDATA[Y]]></ISUNITCOST>
+  <DATEACCT><![CDATA[2017-12-22 00:00:00.0]]></DATEACCT>
+</M_TRANSACTION_COST>
+
+<M_TRANSACTION_COST>